### DISCUSSION TITLE
Has anyone tried to reprogram the $40 Vue version? I don't need all the sensors or the $160 cost

### DISCUSSION THREAD

**Discussion by wannabuy** [emporia-vue-local/esphome#165] [Ideas]
Hey All,
I've got a spare $40 version of the VUE, i'd like to have stream my meter data to a arduino or something i can work with.  
Their 20 minute view time on their app doesn't cut it, and there's no plan on having data permanently streaming on their app.
Has anyone conquered this and got the tee shirt 'been there and done that' with this device?
Thanks,
Bryan

**Comment by flaviut** [comment]
Hey, unfortunately this project is only for the Emporia Vue 2. It will never support Emporia Vue: Utility Connect.

If you'd like, [Home Assistant Glow](https://github.com/klaasnicolaas/home-assistant-glow) has similar functionality to the Emporia Vue: Utility Connect, and should be fairly cheap. I've never used it though, and can't help with it.

**Comment by eigenphase** [comment]
You did not come across https://github.com/jrouvier/esphome-emporia-vue-utility yet?

Works wonderfully for me.
